Dockerfile

```diff
@@ -173,8 +178,12 @@ RUN wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb \
 # Install Rust 1.51.0
 ENV RUSTUP_HOME=/opt/rust \
   PATH="${PATH}:/opt/rust/bin"
+RUN mkdir -p "$RUSTUP_HOME" && \
+  chown -R dependabot:dependabot "$RUSTUP_HOME"
```

Comment:
Do we need this `chown` if we move up the `dependabot` user to before creating the dir?